dicom_utils/visualize.py

```diff
@@ -188,14 +190,31 @@ def draw_rectangle(
     return image
 
 
-def dcms_to_images(dcms: List[Dicom], **kwargs) -> Iterator[DicomImage]:
+def dcms_to_images(dcms: List[Dicom], bar: bool = True, jobs: int = 4, **kwargs) -> Iterator[DicomImage]:
     """Some DICOMs may only contain annotations or other non-image information, so we want to
     pull out image data where applicable."""
-    for dcm in dcms:
+    tqdm_bar = tqdm(desc="Loading images", total=len(dcms), disable=(not bar))
+
+    def func(dcm: Dicom) -> Optional[DicomImage]:
         try:
-            yield DicomImage.from_dicom(dcm, **kwargs)
+            return DicomImage.from_dicom(dcm, **kwargs)
         except Exception as e:
             logger.info(e)
+            return
```

Comment:
You don't need a return statement here since there are no more lines in the function.